### PR TITLE
request permisions for new plugin conjur-simple-integration owner mca…

### PR DESCRIPTION
**Pull request by mcamposv**
https://github.com/jenkinsci/conjur-simple-integration-plugin
https://issues.jenkins-ci.org/browse/HOSTING-952

# Description
https://github.com/jenkinsci/conjur-simple-integration-plugin
https://issues.jenkins-ci.org/browse/HOSTING-952

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [ ] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [ ] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@daniel-beck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

**Comment by mcamposv** [comment]
> " Add link to resolved HOSTING issue in description above". Submitter checklist exists for a reason @mcamposv . It is hard to verify requests without context links
sorry @oleg-nenashev it is everything a bit confusing for the first time.
Actually the new repository was created after a issue I placed on jira. and I have develop permisions there so I do not really know if this pull request is still needed. Could you clarify that.

Thanks.

**Comment by slide** [comment]
If you want to release your plugin to the Jenkins update center, you need this PR.

**Comment by mcamposv** [comment]
> If you want to release your plugin to the Jenkins update center, you need this PR.

Then I definitely need this pull request the hole Idea is to have this plugin published as a jenkins plugins so people can browse it an use easily if they feel it can help them.
 
I have re read the the checklist. and I think I understand better some of the checks.
I think I marked the right ones.
as I mention is a bit complicated when you are doing this for the first time.
I just want publish my plugin to the community so other users can enjoy it.

Can you review and explain me if I'm still missing something?
Thanks.

File: permissions/plugin-conjur-simple-integration.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "conjur-simple-integration"
+github: "jenkinsci/conjur-simple-integration"
+paths:
+- "org/conjur/jenkins/conjur-simple-integration"
```
Comment (slide) [inline permissions/plugin-conjur-simple-integration.yml:5]:
```suggestion
- "io/jenkins/plugins/conjur-simple-integration"
```

This should be the groupId + artifactId in path form see your pom.xml: https://github.com/jenkinsci/conjur-simple-integration-plugin/blob/master/pom.xml#L10

File: permissions/plugin-conjur-simple-integration.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: "conjur-simple-integration"
+github: "jenkinsci/conjur-simple-integration"
```
Comment (slide) [inline permissions/plugin-conjur-simple-integration.yml:3]:
```suggestion
github: "jenkinsci/conjur-simple-integration-plugin"
```

Comment (mcamposv) [reply permissions/plugin-conjur-simple-integration.yml]: > 
Thanks for your update.
I did not realized that that was changed after I changed the name for the hosting request.

**Pull request merged**